fetch.bs

```diff
@@ -8267,6 +8235,16 @@ in a <a for=/>parallel queue</a> if <a for=fetch><i>useParallelQueue</i></a> is
  <dt><a for=fetch><i>useParallelQueue</i></a>
  <dd><p>Takes a <a for=/>boolean</a> that defaults to false. Indicates where the algorithms passed
  as arguments will be invoked. Hopefully most standards will not need this.
+
+ <dt><a for=fetch><i>initiatorType</i></a>
+ <dd><p>Takes a string that defaults to "<code>other</code>", to indicate the
+ {{PerformanceResourceTiming/initiatorType}} for <cite>Resource Timing</cite>. [[RESOURCE-TIMING]]
+
+ <dt><a for=fetch><i>timingGlobal</i></a>
+ <dd><p>Takes "<code>client</code>", "<code>none</code>" or a <a for=/>global object</a> (default
```

Comment:
Should `"<code>none</node>"` be `null`? I see in some text in this change comapres `timingGlobal` with `null`.

Reply:
Right. I also changed the default to be null - this way callers that want to report to resource timing have to be explicit about it (some fetches don't report)